src/components/Radio/RadioGroup.tsx

```diff
@@ -99,6 +99,10 @@ export const RadioGroup: React.FC<RadioGroupProps> = ({
 
   const [contextValue, setContextValue] = React.useState(value);
 
+  React.useEffect(() => {
+    setContextValue(value);
+  }, [value]);
```

Comment:
Hmm, I'm not sure I'm following why the existing code doesn't cover this?  If a radio option in the radio group is clicked, the `handleContextChange` should be called, calling `setContextValue` as well, right?

Reply:
Yes it is. I have a form that I am working with and once the form is reset the radio group is not respecting setting it back to the default value

Reply:
Ah okay, makes sense! Thanks!